src/components/Quote.js

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+import '../css/quote.css';
+
+function Quote() {
+  const [info, setInfo] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      try {
+        const request = await fetch('https://api.api-ninjas.com/v1/quotes?category=education', {
+          method: 'GET',
+          headers: {
+            'X-Api-Key': 'xCLkSI42UwwiL0aUBSRxdg==cm9h7DS0qcaBr11g',
+          },
```

Comment:
- [ ] [OPTIONAL] Api keys are considered to be very sensitive data which you should never commit on git as they could pose a huge security risk. Please consider using environment variables instead. 
You can use this [resource](https://www.architect.io/blog/2022-08-16/react-environment-variables-developers-guide/) as a guide 😎

Reply:
Hi @codepantha,
Thanks so much for that education. I will check out the resource and implement your suggestion accordingly.